packages/ui-components/src/components/InputGroup/input-group.scss

```
@@ -6,8 +6,7 @@
 .juno-input-group {
   // Remove border-radius for specific elements within the input group
   .juno-button,
-  .juno-textinput,
-  .juno-select {
+  .juno-textinput {
```

This can be deleted. It doesn't work, because `juno-textinput` is always wrapped in a parent element
```suggestion
   {
```